pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -270,7 +270,7 @@ public static Properties loadPropertyFiles(String... names) {
 
   public static void initDriver() {
     synchronized (TestUtil.class) {
-      if (initialized) {
+      if (initialized && Driver.isRegistered()) {
```

Comment:
Might be better removing the initialized variable all together?

Reply:
Good idea. "initialized" duplicates with Driver.isRegistered(). Let's rely on driver state.